qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -148,15 +148,32 @@ helm upgrade uaa ${CAP_DIRECTORY}/helm/uaa${CAP_CHART}/ \
 # Wait for UAA namespace
 wait_for_namespace "${UAA_NAMESPACE}"
 
-# Determine the Helm revision of the chart controlling the specified namespace.
-helm_revision() { helm list --date --reverse --max 1 --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
-get_uaa_secret () {
-    kubectl get secret secret-$(helm_revision "${UAA_NAMESPACE}") \
-    --namespace "${UAA_NAMESPACE}" \
-    -o jsonpath="{.data['$1']}"
+
+function semver_is_gte() {
+  # Returns successfully if the left-hand semver is greater than or equal to the right-hand semver
```

Comment:
Returns successfully if the left-hand semver (symmetrical version) is greater than or equal to the right-hand semver

Reply:
Also can u add the logic example you were explaining 10.1.2 vs 2.3.4

Reply:
semantic version.. but this is a well-known thing: https://semver.org/